cpp/roaring64map.hh

```diff
@@ -646,6 +647,44 @@ class Roaring64Map{
     }
 
     /**
+     * Print the content of the bitmap into a string
+     */
+    std::string toString() const {
```

Comment:
I decided to use an STL string for output. C++ users nowadays are either using the STL or a custom string class that we cannot guess at.